xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.xml

```diff
@@ -84,6 +84,10 @@ rendering.macro.menu.parameter.type.name=Type
 rendering.macro.menu.parameter.type.description=The optional menu type determines the menu appearance and behaviour. The supported values are: horizontal (default value) and vertical.
 rendering.macro.menu.content.description=Define the menu structure using wiki syntax. Each menu item should be a list item and should contain the menu item label or link. You can use nested lists for sub-menu items.
 
+# Accessibility labels
+rendering.macro.menu.horizontal.label=Top navigation menu: {0}
+rendering.macro.menu.vertical.label=Side navigation menu: {0}
```

Comment:
I'm not sure this `rendering.macro` prefix should be used here, this is specific for the macro and parameter names and descriptions. See also https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/#HKeyformat for the general naming rules.